cmd/autonity/chaincmd.go

```diff
@@ -215,30 +199,49 @@ Use "ethereum dump 0" to dump the genesis block.`,
 // initGenesis will initialise the given JSON format genesis file and writes it as
 // the zero'd block (i.e. genesis) or will fail hard if it can't succeed.
 func initGenesis(ctx *cli.Context) error {
-	// Make sure we have a valid genesis JSON
-	genesisPath := ctx.Args().First()
+	// If user does not specify a genesis file on genesis flag, start node from current data dir.
+	if !ctx.GlobalIsSet(utils.InitGenesisFlag.Name) {
+		log.Info("genesis flag is not set, start node directly from data dir.")
+		log.Info("In case of 1st time start, please start client with setting --genesis flag, " +
+			"the client will auto init genesis block before attach to network.")
+		return nil
+	}
```

Comment:
I don't think there is a need for the second log statement as the info below will provide the same information. Also instead of saying `start node directly from data dir` in the first log statement is confusing because the user doesn't know which data dir is being reffered.

```suggestion
	if !ctx.GlobalIsSet(utils.InitGenesisFlag.Name) {
		log.Info("--genesis flag is not set")
		return nil
	}
```